package.json

```diff
@@ -41,11 +41,12 @@
     "gulp-sass": "^5.1.0",
     "gulp-uglify": "^3.0.2",
     "markdown-it-anchor": "^9.0.1",
+    "nhsuk-frontend": "^8.1.1",
     "nunjucks": "^3.2.4",
     "prettier": "^3.2.5",
     "sass": "^1.74.1"
   },
-  "dependencies": {
+  "peerDependencies": {
```

Comment:
Should this peerDependency have a version like "8.x" (or "^8.0.0")?
There's nothing specific in 8.1.1 that we must require our users to have.

Reply:
We could make it `^8.0.0` yeh as this will just install `8.2.0`. The idea being we are betting on MINOR bumps not to affect anything for us.